src/context/schemas/action.schema.json

```
@@ -0,0 +1,26 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://fdc3.finos.org/schemas/next/action.schema.json",
+  "type": "object",
+  "title": "Action",
+  "properties": {
+    "type": { "const": "fdc3.entity.action" },
+    "title": {
+      "type": "string"
+    },
+    "intent": {
+      "type": "string",
+      "$comment": "Should reference one of the intent types listed in 'standard intents'"
```

```suggestion
      "$comment": "Should reference an intent type name, such as those defined in the FDC3 Standard'"
```

Could be used with arbitrary intent types